index.html

```diff
@@ -705,7 +705,9 @@ <h2 id="AudioContext">
           <dd>
             <p>
               This is a time in seconds which starts at zero when the context
-              is created and increases in real-time. All scheduled times are
+              is created and increases in real-time. At any given point, its value
+              represents the starting time of the next block of audio to be processed
```

Comment:
@padenot @rtoy Could you please suggest alternate wording? The definition above is my best understanding of what is actually implemented so I need your help here.